H2MLauncher.Core/Settings/ResourceSettings.cs

```diff
@@ -10,7 +10,8 @@ public record IW4MObjectMap(string Name, string Alias);
 
     public class IW4MMapPack()
     {
-        public string Name { get; set; } = string.Empty;
-        public List<IW4MObjectMap> Maps { get; set; } = [];
+        public required string Name { get; init; }
+        public required string Id { get; init; }
```

Comment:
Tbf, the `Name` suffices as `Id` here. Why add it?

Reply:
So we have the option to change and rename map packs later